### PR TITLE
Suggested Route Table

### PR DESCRIPTION
**Pull request by lil2400**
Had to delete the old PR #93 due to some git messed up things.

### CODE REVIEW

File: src/client/components/SelectedTable.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              </TableCell>
+              <TableCell align="right">{row.latitude}</TableCell>
+              <TableCell align="right">{row.longitude}</TableCell>
+              <TableCell align="right">{row.speed}</TableCell>
+              <TableCell align="right">{row.estimated_rpm}</TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+      <h4>Currently optimizing for the cheapest possible route.</h4>
```
Comment (zkwsk) [inline src/client/components/SelectedTable.js]: You need to expose this as a prop so we can display different text based on what the optimisation setting is.

File: src/client/components/SelectedTable.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "@storybook/addon-knobs/register";
+
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+
+function SelectedTableComponent({ data = [] }) {
```
Comment (zkwsk) [inline src/client/components/SelectedTable.js]: I would like if you can call the component SuggestedRouteTable

File: src/client/stories/SuggestedTableStory.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "@storybook/addon-knobs/register";
+import SelectedTableComponent from "../components/SelectedTable";
+
+import { object } from "@storybook/addon-knobs";
+
+export function SuggestedTableStory() {
```
Comment (zkwsk) [inline src/client/stories/SuggestedTableStory.js]: SuggestedRouteTableStory

File: src/client/stories/SuggestedTableStory.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    {
+      date: "2019-01-01 09:08:50",
+      latitude: 28.650932,
+      longitude: 86.435767,
+      speed: 19,
+      estimated_rpm: 61
+    }
+  ]);
+  return <SelectedTableComponent data={data} />;
```
Comment (zkwsk) [inline src/client/stories/SuggestedTableStory.js]: Call it SuggestedRouteTable

**Review by zkwsk — CHANGES_REQUESTED** [review]
Hey Ilir. Good work. Needs a few small fixes. Also, I merged the latest develop and fixed conflicts and added package dependency for you.

File: src/client/components/SelectedTable.js (hunk 1 above)
Comment (lil2400) [reply src/client/components/SelectedTable.js]: Do you mean the h4?

**Review by Swapna-Sahu — DISMISSED** [review]
looks good to me

**Review by Swapna-Sahu — APPROVED** [review]
Looks good to me

**Review by zkwsk — APPROVED** [review]
Looks good 👍

**Pull request merged**